src/pod.c

```diff
@@ -105,6 +120,35 @@ add_container_mount(struct cc_oci_config *config) {
 	return false;
 }
 
+/**
+ * Returns a pod namespace ID from an OCI annotation.
+ *
+ * \param annotation \ref oci_cfg_annotation.
+ *
+ * \return a valid pod namespace ID on success, and CC_POD_INVALID on failure.
+ */
+private enum pod_namespace_id
+pod_namespace_present(struct oci_cfg_annotation *annotation)
+{
+	if (annotation == NULL || annotation->key == NULL) {
+		return CC_POD_INVALID;
+	}
+
+	/* We only handle CRI-O and ocid annotations for now */
+	/* Let's check for CRI-O first */
+	if (strncmp(annotation->key, CC_POD_CRIO_NAMESPACE,
+		    CC_POD_CRIO_NAMESPACE_SIZE) == 0) {
```

Comment:
that macro is a string literal. What about to use sizeof(CC_POD_CRIO_NAMESPACE) to get the length of that in the case that change in the future we wont forget change that value in the future?